### PR TITLE
Test swap fee helpers

### LINKED ISSUE #1390: [Swap] Test latest helpers of `Swap.utils`
- `minAmountToSwapMax1e8`
- `minBalanceToSwap`

^ All added by #1394

### PR DESCRIPTION
**Pull request by veado**
All tests in ` Swap.utils`

- [x] Test `minBalanceToSwap` + `calcRefundFee` + `minAmountToSwapMax1e8`
- [x] Remove `refundFees` from `SwapFees` in favour of `calcRefundFee` helper

Closes #1390

### CODE REVIEW

File: src/renderer/components/swap/Swap.utils.test.ts [hunk 1]
```diff
@@ -293,4 +298,241 @@ describe('components/swap/utils', () => {
...
+      const result = minBalanceToSwap(params)
+      expect(eqBaseAmount.equals(result, baseAmount(600))).toBeTruthy()
+    })
+  })
+
+  describe('minAmountToSwapMax1e8', () => {
+    const poolsData = {
+      'BNB.BUSD-74E': {
+        assetBalance: assetToBase(assetAmount(20)), // 1 BUSD = 0.05 RUNE
+        runeBalance: assetToBase(assetAmount(1)) // 1 RUNE = 20 BUSD
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/swap/Swap.utils.test.ts:326]: today.... i love the idea `1 rune = 20 usd` 😅

File: src/renderer/components/swap/Swap.utils.test.ts [hunk 2]
```diff
@@ -293,4 +298,241 @@ describe('components/swap/utils', () => {
...
+      // inboundFeeInBUSD + outboundFeeInBUSD
+      // 0.0001 * 600 + 0.0003 * 600 = 0.06 + 0,18 = 0.24
+      //
+      // Formula (failure):
+      // inboundFeeInBUSD + refundFeeInBUSD
+      // 0.0001 * 600 + 0.0003 * 600 = 0.06 + 0,18 = 0.24
+      //
+      // Formula (minValue):
+      // 1,5 * max(success, failure)
+      // 1,5 * max(0.24, 0.24) = 1,5 * 0.24 = 0.36
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/swap/Swap.utils.test.ts:381]: good idea to have this comments here 👍

File: src/renderer/components/swap/Swap.utils.test.ts [hunk 3]
```diff
@@ -293,4 +298,241 @@ describe('components/swap/utils', () => {
...
+        runeBalance: assetToBase(assetAmount(30)) // 1 RUNE = 0.03 BNB
+      },
+      'ETH.ETH': {
+        assetBalance: assetToBase(assetAmount(1)), // 1 ETH = 100 RUNE (2000 USD)
+        runeBalance: assetToBase(assetAmount(100)) // 1 RUNE = 0.01 ETH
+      }
+    }
+
+    it('non chain asset -> chain asset (same chain): BNB.USD -> BNB.BNB)', () => {
+      const inAssetDecimal = 6
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/swap/Swap.utils.test.ts:343]: maybe just BNB_DECIMAL ?

**Review by thatStrangeGuyThorchain — APPROVED** [review]

File: src/renderer/components/swap/Swap.utils.test.ts (hunk 3 above)
Comment (veado) [reply src/renderer/components/swap/Swap.utils.test.ts:343]: _I think it's ok to add such "mock" value for testing..._

**Pull request merged**